Sources/Apodini/Semantic Model Builder/OpenAPI/Extensions/AnyEndpointParameter+OpenAPI.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Created by Lorena Schlesinger on 09.12.20.
+//
+
+import Foundation
+@_implementationOnly import OpenAPIKit
+
```

Comment:
```suggestion
extension OpenAPI.Parameter.Context {
    init?(_ endpointParameter: AnyEndpointParameter) {
        switch endpointParameter.parameterType {
        case .lightweight:
            self = .query
        case .content:
            return nil
        case .path:
            self = .path
        }
    }
}
```

This is just a suggestion, but I worry, that core types of Apodini can get polluted with extensions that are built on top of them and don't reflect their inherent functionality.

This approach should deflect the growing duty on these core Apodini types.

What do you think?

Reply:
Absolutetly right, thanks for the suggestion!! Refactored in 5e54667